CHARTER.adoc

```diff
@@ -0,0 +1,195 @@
+:Title: OGC GeoZarr Standards Working Group Charter
+:titletext: {Title}
+:doctype: book
+:encoding: utf-8
+:lang: en
+:toc:
+:toc-placement!:
+:toclevels: 4
+:numbered:
+:sectanchors:
+:source-highlighter: pygments
+
+<<<
+[cols = ">",frame = "none",grid = "none"]
+|===
+|{set:cellbgcolor:#FFFFFF}
+|[big]*Open Geospatial Consortium*
+|Submission Date: 2023-05-01
+|Approval Date: 
+|Internal reference number of this OGC(R) document: 23-xxx
+|Category: OGC(R) Standards Working Group Charter
+|Authors: Christophe Noel, Brianna R. Pagán
+|===
+
+[cols = "^", frame = "none"]
+|===
+|[big]*{titletext}*
+|===
+
+[cols = "^", frame = "none", grid = "none"]
+|===
+|*Copyright notice*
+|Copyright (C) 2021 Open Geospatial Consortium
+|To obtain additional rights of use, visit http://www.opengeospatial.org/legal/
+|===
+
+<<<
+
+To: OGC members & interested parties
+
+A new OGC Standards Working Group (SWG) is being formed. The OGC members listed below have proposed the OGC GeoZarr SWG.  The SWG proposal provided in this document meets the requirements of the OGC Technical Committee (TC) Policies and Procedures.
+
+The SWG name, statement of purpose, scope, list of deliverables, audience, and language specified in the proposal will constitute the SWG's official charter. Technical discussions may occur no sooner than the SWG's first meeting.
+
+This SWG will operate under the OGC IPR Policy. The eligibility requirements for becoming a participant in the SWG at the first meeting (see details below) are that:
+
+* You must be an employee of an OGC member organization or an individual
+member of OGC;
```

Comment:
I'm pretty uncomfortable with this requirement. Can we say instead that any interested party can join the swg?

Reply:
No that's not possible. However, Scott already explained how interested parties can attend the meetings and contribute after signing an administrative paper.

Reply:
Voting SWG members must be OGC members but OGC activities can be inclusive such that a person is a SWG contributor. Perhaps clarifying the distinction between member and contributor would be useful?

Reply:
This text is standard letter of the OGC (see for example https://portal.ogc.org/files/96156), so I don't know if we can change this part.